### ISSUE TITLE
WIFI does not connect on remote speaker after last update

### ISSUE THREAD

**Issue by itnassol** [esphome/issues#6370]
### The problem

I use round ESP boards based on the Onju Voice, been working beautifully up until the last update for ESP, The speakers now just get into a loop where they are constantly connecting to WIFI but not getting anywhere.

### Which version of ESPHome has the issue?

2024.10.1

### What type of installation are you using?

Home Assistant Add-on

### What platform are you using?

ESP8266

**Comment by itnassol** [comment]
So this seems to be an API issue, if I reboot any of the speakers it wont connect to the API... Any help would be great

**Comment by randybb** [comment]
First format it right (use the code block as this is unreadable) and provide serial logs

**Comment by itnassol** [comment]
Yep you're right my bad, rushed it (one of those days) Sorry all formatted now, Logs in the right place... Am fairly new to thi, what are serial logs? Thanks for giving me a prod.

**Comment by randybb** [comment]
Finally it is readable :) Thanks.

But serial logs - these one are just telling that the dashboard it is not able to connect to address 192.168.68.52 as port 6053 is not open.

Since it is S3 you can just connect to one of USB ports and check logs when it boots. If it is boot loping we might figure out why it is doing so.

**Comment by ShohninDmitriy** [comment]
WIFI logging has been broken again
[logs_jbd-bms_logs_usb.txt](https://github.com/user-attachments/files/17498134/logs_jbd-bms_logs_usb.txt)
[logs_jbd-bms_logs_wifi.txt](https://github.com/user-attachments/files/17498137/logs_jbd-bms_logs_wifi.txt)

**Comment by randybb** [comment]
@ShohninDmitriy this is absolutely unrelated to this issue - as you see, your device is working just fine, your esphome dashboard is broken, probably something was updated and is not compatiblke anymore (rebuild addon/docker/venv).

**Comment by ShohninDmitriy** [comment]
> @ShohninDmitriy this is absolutely unrelated to this issue - as you see, your device is working just fine, your esphome dashboard is broken, probably something was updated and is not compatiblke anymore (rebuild addon/docker/venv).

After reinstalling, alas, it does not work either

**Comment by Jansen-AL** [comment]
Same problem with Respeaker lite kit ans Atom echo. No wifi connection anymore.

**Comment by itnassol** [comment]
The latest update seems to have fixed the problem for me, thank you for all the advice.
